scala-package/native/src/main/native/org_apache_mxnet_native_c_api.cc

```diff
@@ -892,6 +892,119 @@ JNIEXPORT jint JNICALL Java_org_apache_mxnet_LibInfo_mxExecutorBackward
   return ret;
 }
 
+JNIEXPORT jint JNICALL Java_org_apache_mxnet_LibInfo_mxExecutorReshape
```

Comment:
Is that possible to partially migrate the implementation to the Scala side for maintenance? It seemed too complicated for this method.

Reply:
@yzhliu

Reply:
as discussed offline with @lanking520 , the code before and after `MXExecutorReshape` are for converting Java object to C++ object (e.g., string) and vice versa. Thus have to use jni to do so.